Microsoft.DotNet.Interactive/ExtensionLoader.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public static class ExtensionLoader
+    {
+        public static async Task LoadFromAssembly(Assembly assembly, IKernel kernel)
```

Comment:
This can be `internal`.

Reply:
If we make this internal, how will it be accessible from the tests